toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -2,8 +2,9 @@
 
 ## Overview
 
-Given a full disk image, the Azure Linux Image Customizer can generate a
-LiveOS ISO image when the `--output-image-format` is set to `iso`.
+Given a full disk image generated by the Azure Linux toolkit, the Azure Linux
```

Comment:
We should avoid mentioning the "toolkit" in the docs. It is probably better to say something like: official Azure Linux images.

Reply:
Does MIC support images that are built by the user using the Azure Linux toolkit though? Those are not official Azure Linux image... but are using official tooling. I am concerned that if I change it to 'Official Azure Linux images', it would mean that we do not support images created by the official tooling (unless that's really the case).

Reply:
It is probably possible to create images using the toolkit that will confuse MIC. So, it is probably best to not say we explicitly support all images created by the toolkit. (That being said, we don't need to explicitly say we don't support custom images either.)